package/kedro_viz/api/graphql.py

```diff
@@ -240,16 +241,21 @@ def get_run_tracking_data(
         all_runs = {}
         for run_id in run_ids:
             run_id = ID(run_id)
-            file_path = dataset._get_versioned_path(str(run_id))
-            if Path(file_path).is_file():
+            dataset._version = DatasetVersion(run_id, None)
```

Comment:
What does this line do exactly? Is it to set the `load_version` to the `run_id`? I think a short note on what's happening here would be useful for future reference.